src/main/scala/se/nimsa/dcm4che/streams/DicomPartFlow.scala

```diff
@@ -156,6 +156,13 @@ class DicomPartFlow(chunkSize: Int = 8192, stopTag: Option[Int] = None, inflate:
         }
         parseResult
       }
+      override def onTruncation(reader: ByteReader): Unit =
+        if (reader.hasRemaining)
+          super.onTruncation(reader)
+        else {
+          emit(objOut, DicomValueChunk(state.bigEndian, ByteString.empty, last = true))
```

Comment:
This is where we emit the empty chunk if the stream stopped and there were no more bytes to parse.